spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -181,6 +186,14 @@ def get_parser():
              "centerline and the axial plane. If the cord is already quasi-orthogonal to the slab, you can set "
              "-angle-corr to 0."
     )
+    optional.add_argument(
+        '-angle-corr-centerline',
+        metavar=Metavar.str,
+        help="Centerline to be used for angle correction. This is optional; if not provided, the centerline will be "
```

Comment:
Should it be the spinal cord centerline or spinal cord segmentation file that we should provide here? Can it be either?

Reply:
It can be either! Good point, yes, that isn't clear. :)